### PR TITLE
SDIT-1852 Casenotes migration

### PR DESCRIPTION
**Pull request by steverendell**
Initial commit with dummy DPS casenotes service

### CODE REVIEW

File: helm_deploy/hmpps-prisoner-from-nomis-migration/values.yaml [hunk 1]
```diff
@@ -135,6 +141,10 @@ generic-service:
       HMPPS_SQS_QUEUES_EVENTALERTS_QUEUE_NAME: "sqs_name"
     prison-events-alerts-dl-queue:
       HMPPS_SQS_QUEUES_EVENTALERTS_DLQ_NAME: "sqs_name"
+#    prison-events-casenotes-queue:
+#      HMPPS_SQS_QUEUES_EVENTCASENOTES_QUEUE_NAME: "sqs_name"
+#    prison-events-casenotes-dl-queue:
+#      HMPPS_SQS_QUEUES_EVENTCASENOTES_DLQ_NAME: "sqs_name"
```
Comment (steverendell) [inline helm_deploy/hmpps-prisoner-from-nomis-migration/values.yaml:147]: These queues dont exist yet

File: .circleci/config.yml [hunk 2]
```diff
@@ -37,6 +37,7 @@ jobs:
 
   tests:
     parallelism: << pipeline.parameters.test-executors-count >>
+    resource_class: large
```
Comment (steverendell) [inline .circleci/config.yml:40]: I was getting OOM errors when i first created the PR

File: helm_deploy/values-prod.yaml [hunk 3]
```diff
@@ -19,6 +19,8 @@ generic-service:
     API_BASE_URL_ACTIVITIES: https://activities-api.prison.service.justice.gov.uk
```
Comment (andymarke) [inline helm_deploy/values-prod.yaml:19]: I think you are missing feature switches for the CASENOTE tag events for prod?

**Review by andymarke — APPROVED** [review]
LGTM

File: helm_deploy/values-prod.yaml (hunk 3 above)
Comment (steverendell) [reply helm_deploy/values-prod.yaml:19]: Good point, will add next time .. im sure there will be another PR needed once i try a migration in dev

**Pull request merged**